Makefile

```diff
@@ -66,9 +67,12 @@ protos: lint-protos ## Generate Go pb and grpc binding for .proto files
 		-I proto \
 		--go_out=paths=source_relative:pb \
 		--go-grpc_out=paths=source_relative:pb \
-		$(shell find proto -name google -prune -o -regex '.*/[^_].*\.proto' -print)
+		$(PROTOFILES)
 	gofumpt -w .
 
+pb: lint-protos ## Generate binary FileDescriptorSet output pb file
+	protoc -I proto -o proto/all.pb --include_imports $(PROTOFILES)
```

Comment:
Feels wrong to have a generated binary in the `proto` directory. I assume this will be used for testing? Perhaps we can generate it into a `testdata` directory?

Reply:
ok. done.